assembl/locale/de/LC_MESSAGES/assembl.po

```diff
@@ -268,83 +287,140 @@ msgstr "E-Mail-Domain erforderlich"
 
 #: assembl/models/preferences.py:730
 msgid ""
-"List of domain names of user email address required for self-registration. "
-"Only accounts with at least an email from those domains can self-register to "
-"this discussion. Anyone can self-register if this is empty."
+"List of domain names of user email address required for self-"
+"registration. Only accounts with at least an email from those domains can"
+" self-register to this discussion. Anyone can self-register if this is "
+"empty."
 msgstr ""
-"Liste der für die Selbstregistrierung erforderlichen Domainnamen von Benutzer-E-Mail-Adressen. Nur Konten mit einer E-Mail aus diesen Domänen können sich für diese Diskussion selbst registrieren. Anyone can self-register if this is empty."
+"Liste der für die Selbstregistrierung erforderlichen Domainnamen von "
+"Benutzer-E-Mail-Adressen. Nur Konten mit einer E-Mail aus diesen Domänen "
+"können sich für diese Diskussion selbst registrieren. Anyone can self-"
+"register if this is empty."
 
 #: assembl/models/preferences.py:746
+#, fuzzy
+msgid "Whitelist on authentication service"
+msgstr "Diensttyp für Authentifizierung"
+
+#: assembl/models/preferences.py:749
+msgid "Allow white list to be applied to the authentication service chosen"
+msgstr ""
```

Comment:
Can you work with @MilanBarande here to fill in the German for these strings? Update the `msgstr` to be the German translation of what the `msgid` is. Thanks!